src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -188,6 +188,10 @@ var DOMPropertyOperations = {
     }
     if (value == null) {
       node.removeAttribute(name);
+    } else if (value === true) {
+      node.setAttribute(name, '');
+    } else if (value === false) {
+      node.removeAttribute(name);
```

Comment:
E.g. `spellcheck` is not a boolean attribute and expects the value `'true'` or `'false'`, if anyone relies on boolean being stringified today it will break for them (I assume? or is that another codepath). Also note that neither `'true'` nor `'false'` is the default value for it.